WordPress/Sniffs/XSS/EscapeOutputSniff.php

```diff
@@ -152,6 +175,24 @@ public function process_token( $stackPtr ) {
 			if ( in_array( $function, array( 'trigger_error', 'user_error' ), true ) ) {
 				$end_of_statement = $this->phpcsFile->findEndOfStatement( $open_paren + 1 );
 			}
+		} else if ( false === $this->short_open_tag_enabled && T_INLINE_HTML === $this->tokens[ $stackPtr ]['code'] ) {
```

Comment:
Just noticed one last minor thingie: the first part of this `if` condition is not necessary as `T_INLINE_HTML` is only registered to this sniff if `short_open_tag_enabled` is `true`.

Having said that, that also means that the property can be removed and turned into a local variable within the `register()` method.